### ISSUE TITLE
Please provide QT5 build option for Linux

### ISSUE THREAD

**Issue by fcahoon** [ccrma/miniAudicle#40]
Apps built with QT 5.6 or above will scale automatically so they are usable on linux with hidpi displays. When I build using the provided instructions (cd src ; make linux-jack), the code is linked with QT4, which makes the app render very tiny for me.

It shouldn't be hard to just link with QT5, right? Please provide that as an option, or just move to QT5.

**Comment by spencersalazar** [comment]
Interesting, thats not so good that it renders tiny, we can look into it. 

Unfortunately my experience is that major library upgrades are not so simple as there are often compatibility issues at the source code level and functionality level. Its not really possible to gauge the difficulty without diving into it outright.

**Comment by fcahoon** [comment]
The workaround is to use [run_scaled](https://github.com/kaueraal/run_scaled), so it's not a showstopper for me. I don't get the crisp resolution my screen is capable of providing that way, but it will do.

**Comment by ycollet** [comment]
Just as a comment: Fedora 32 doesn't ship qt4 anymore. So, impossible to build miniAudicle on Fedora 32.

**Comment by mariobuoninfante** [comment]
same with Ubuntu Studio 20.04 LTS

**Comment by ycollet** [comment]
To be more precise: on fedora, they have renamed qt to qt4 but the big problem is qscintilla. The Qt4 version is not shipped anymore.
I have rebuilt the qscintilla-qt4 package on my machine and miniAudicle compiles fine (with gcc 10).

**Comment by fgaz** [comment]
On NixOS, too, we'll have to drop the miniaudicle package due to lack of qt5 support https://github.com/NixOS/nixpkgs/pull/174634

**Comment by fgaz** [comment]
Looks like miniaudicle was updated straight to qt6:

https://github.com/ccrma/miniAudicle/blob/e6d587ce4126a625e8c2c3c9089d64b16e9f7a11/README.md?plain=1#L35

Can we get a release that includes the update?

**Comment by gewang** [comment]
> Can we get a release that includes the update?

1.5.0.3 just came out: https://chuck.stanford.edu/release/. We've made a [miniAudicle release source tarball](https://chuck.stanford.edu/release/files/miniAudicle-1.5.0.3.tgz) for this release (with 'chuck' dependency included). The updated build instructions can be found at https://github.com/ccrma/miniAudicle#readme

Please let us know how we can help (the install targets may need some TLC). Two things regarding potential packaging/installation:
1) the examples folder should somehow be installed so it's accessible from miniAudicle->File->Open Examples...
2) the tarball also contains src/chugins (another repo containing chuck plugins, most of which ship with miniAudicle; these typically go into /usr/local/lib/chuck/, one of the default search paths for chuck/miniAudicle to autoload chugins, when starting the VM)

I will help as I can! but @spencersalazar holds the real knowledge of the miniAudicle...

btw if you Discord, the [ChucK Community Discord server](https://discord.gg/ENr3nurrx8) might be helpful for interactive communications as well. Thank you again!

**Comment by fgaz** [comment]
Sorry, I almost forgot about this. So does this mean that `chuck-*` tags in this repo are to be considered miniaudicle releases?

**Comment by gewang** [comment]
Yes, miniAudicle and CLI chuck releases are synchronized and both repos use the same chuck-* format to tag their respective release. The latest presently should be `chuck-1.5.1.0`, with `chuck-1.5.1.1` around the corner (ETA next week)
